manifests/init.pp

```diff
@@ -12,15 +12,44 @@
 #   If true, ensure policycoreutils-python is installed. This is a supplemental
 #   package that is required by semanage.
 #
+# @param manage_mcstrans_package
+#   Manage the `mcstrans` package.
+#
+# @param manage_mcstrans_service
+#   Manage the `mcstrans` service.
+#
+# @param mcstrans_service_name
+#  The `mcstrans` service name.
+#
+# @param mcstrans_package_name
+#  The `mcstrans` package name.
+#
+# @param manage_restorecond_package
+#   Manage the `restorecond` package.
+#
+# @param manage_restorecond_service
+#   Manage the `restorecond` service.
+#
+# @param restorecond_package_name
+#   The `restorecond` package name.
+#
+# @param package_ensure The ensure status of packages to be installed
+#
 class selinux (
   Selinux::State         $ensure               = simplib::lookup('simp_options::selinux', { 'default_value' => true }),
   Boolean                $manage_utils_package = true,
   String                 $package_ensure       = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
-  Enum['targeted','mls'] $mode                 = 'targeted'
+  Enum['targeted','mls'] $mode                 = 'targeted',
+  # defaults are in module data
+  Boolean $manage_mcstrans_package,
```

Comment:
These will need to be pushed to the top of the stack so that the linter doesn't complain.